library/src/main/scala/scala/scalajs/runtime/StackTrace.scala

```diff
@@ -153,9 +153,9 @@ object StackTrace {
    *  display the function name.
    */
   private def extractClassMethod(functionName: String): (String, String) = {
-    val PatC = """^ScalaJS\.c\.([^\.]+)(?:\.prototype)?\.([^\.]+)$""".re
-    val PatI = """^(?:Object\.)?ScalaJS\.i\.((?:_[^_]|[^_])+)__([^\.]+)$""".re
-    val PatM = """^(?:Object\.)?ScalaJS\.m\.([^.\.]+)$""".re
+    val PatC = """^(?:ScalaJS\.c\.|\$c_)([^\.]+)(?:\.prototype)?\.([^\.]+)$""".re
+    val PatI = """^(?:Object\.)?(?:ScalaJS\.s\.|\$s_)((?:_[^_]|[^_])+)__([^\.]+)$""".re
+    val PatM = """^(?:Object\.)?(?:ScalaJS\.m\.|\$m_)([^.\.]+)$""".re
```

Comment:
Hum, do we have anything that allows to run a test only in FastOpt (but not in FullOpt)? In FullOpt all of this stuff does not work because everything is renamed anyway.